x-pack/plugins/security_solution/public/flyout/document_details/right/components/graph_preview_container.tsx

```diff
@@ -29,6 +60,9 @@ export const GraphPreviewContainer: React.FC = () => {
     ecsData: dataAsNestedObject,
   });
 
+  const shouldShowGraphPreview =
+    visualizationInFlyoutEnabled && isGraphFeatureEnabled && isAuditLog;
+
```

Comment:
you already have this logic in the parent `visualizations_section` component (line 47 to 49) and you're already preventing this component to render on line 65. I think you should be able to remove the logic either here or in the parent component?